app/views/categories/index.html.erb

```diff
@@ -0,0 +1,54 @@
+<div class='categories-container'>
+<div class='sign-up-details category-header'>
+<%= image_tag 'hamburger.png', class: 'hamburger' %>
+<h3>categories</h3>
+<%= image_tag 'search.png', class: 'search' %>
+</div>
+<div class='card-main-container'>
+<% if @categories.empty? %>
+<h3 class='centered-text'> Add a category </h3>
+<% else %>
+<% @categories.each_with_index do |category, index| %>
+  <a href='/categories/<%= category.id %>/purchases'>
+  <div class='card-container'>
+  <% if category.present? %>
+  <div class='icon-container'>
+  <%= image_tag(category.icon, class: 'category_icon') %>
+  </div>
+  <div class='company-date'>
+  <p><%= category.name %></p>
+  <p class='date'><%= category.created_at.strftime('%e %b %Y') %></p>
+  </div>
+  <p>$ <%= category.total_amount %></p>
+  <% else %>
+  <p>Add a category</p>
+  <% end %>
+  </div>
+  </a>
+<% end %>
+<% end %>
+</div>
+<a href="<%= new_category_path %>" class='add-item'>
+<div class='inner'>
```

Comment:
- [ ] Please make sure that the button spans across horizontally like in the original design.
### Your design
![image](https://github.com/emmanuelsteven/BudgetApp/assets/45460192/8a7393ab-828e-4a4c-ba0d-25a56c21f6a8)
### Required
![image](https://github.com/emmanuelsteven/BudgetApp/assets/45460192/db975b9b-cdbd-4a5a-b862-20272feb3400)